mempool/policy_test.go

```diff
@@ -101,8 +102,9 @@ func TestCalcMinRequiredTxRelayFee(t *testing.T) {
 func TestCheckPkScriptStandard(t *testing.T) {
 	var pubKeys [][]byte
 	for i := 0; i < 4; i++ {
-		pk := chainec.Secp256k1.NewPrivateKey(big.NewInt(int64(chainec.ECTypeSecp256k1)))
-		pubKeys = append(pubKeys, chainec.Secp256k1.NewPublicKey(pk.Public()).SerializeCompressed())
+		pk := secp256k1.NewPrivateKey(big.NewInt(int64(chainec.ECTypeSecp256k1)))
+		pubKeys = append(pubKeys,
+			secp256k1.NewPublicKey(pk.Public()).SerializeCompressed())
```

Comment:
Recall from the previous PR that there is no need to decompose it into X and Y and create a new public key from it.  Plus, `.Public()` is going to be removed.  Just cast it.